package-lock.json

```diff
@@ -1,8 +1,8405 @@
 {
   "name": "pay-with-planet",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Switching to format 2 with npm 7.x for this project is fine, but I need to have npm 6.x and npm 7.x installed on my machine to test now as we are still bound to npm 6.x generating format 1 for the native apps.

Reply:
I don't know how to revert this. If you know how to do it, can you please do it in this PR?

Reply:
I think I did it because the project was not building on my device